localization/i18n/es/OrcaSlicer_es.po

```diff
@@ -2706,7 +2712,8 @@ msgstr "Enviando la configuración de impresión"
 #, c-format, boost-format
 msgid "Successfully sent. Will automatically jump to the device page in %ss"
 msgstr ""
-"Envío exitoso. Se saltará automaticamente a la página del dispositivo en %ss"
+"Envío exitoso. Se alternará automáticamente a la página del dispositivo en "
```

Comment:
No need to add a space before the line jump

Reply:
Done.